extensions/sql-database-projects/src/models/project.ts

```diff
@@ -1442,6 +1476,82 @@ export class Project implements ISqlProject {
 		}
 	}
 
+	/**
+	 * Adds a value to project property, separated by semicolon if
+	 * one or more values are already present in project property
+	 * @param propertyName Name of property
+	 * @param valueToAdd Value to be added to project property, values containing semicolon are not supported
+	 * @param caseSensitive Determines if value to add to collection should be compared against existing values case sensitively
+	 */
+	private async addValueToCollectionProjectProperty(propertyName: string, valueToAdd: string, caseSensitive: boolean = false): Promise<void> {
+		if (valueToAdd.includes(';')) {
+			throw new Error(constants.invalidProjectPropertyValueProvided(valueToAdd));
+		}
+
+		let propertyValueToSet = this.evaluateProjectPropertyValue(propertyName);
+		if (propertyValueToSet === undefined) {
+			propertyValueToSet = valueToAdd;
+		} else { // Property already exists, append to existing value
+			const propertyValue = caseSensitive ? propertyValueToSet : propertyValueToSet.toLocaleLowerCase();
```

Comment:
I don't think the comparison should be locale-specific. Although it probably won't matter as you will lower-case both on the same machine, but using invariant lower/upper will probably be a little bit more reliable (albeit mostly in theory).

Reply:
Hm.. in the `remove` method you actually used non-locale-specific transformations. Is there a reason for the discrepancy?

Reply:
I didn't mean for this discrepancy; it should be consistent between the two functions.  I can keep them as `.toLowerCase()`.